packages/hdwallet-ledger/src/ethereum.ts

```diff
@@ -84,22 +123,26 @@ export function ethSupportsNativeShapeShift(): boolean {
   return false
 }
 
-export function ethGetAccountPaths (msg: ETHGetAccountPath): Array<ETHAccountPath> {
+export function ethGetAccountPaths (msg: core.ETHGetAccountPath): Array<core.ETHAccountPath> {
+  /* TODO support both ETH account paths
```

Comment:
You might have to comment out a corresponding test case.

Reply:
I added one to just test the single path and added skip to the other one until the non bip44 is supported.

Reply:
👍

Reply:
Decided to leave the tests and `ethGetAccountPaths` alone (including both ledger live and chrome derivation paths). I will just add the length to the `ethGetPublicKeys` filter to only include BIP44 account path for now.

Reply:
even better